.github/workflows/build.yaml

```
@@ -143,3 +146,47 @@ jobs:
           APICULTURIST_GITHUB: ${{ secrets.APICULTURIST_GITHUB }}
           APICULTURIST_TOKEN: ${{ secrets.APICULTURIST_TOKEN }}
           monorepo: ${{ inputs.monorepo }}
+
+  dev_build:
+    runs-on: ubuntu-latest
+    needs: read_package
+    name: ${{ matrix.repos }}
+    strategy:
```

Based on read-package output suggestion
```suggestion
    strategy: ${{fromJson(needs.read_package.outputs.strategy)}}
```